android/quest/src/main/assets/configs/app/profiles/default_profile_config.json

```diff
@@ -27,6 +27,35 @@
       }
     ]
   },
+  "secondaryResources": [
+    {
+      "resource": "Group",
+      "dataQueries": [
+        {
+          "id": "commoditiesQueryByCode",
+          "filterType": "TOKEN",
+          "key": "code",
+          "valueType": "CODEABLECONCEPT",
+          "valueCoding": {
+            "system": "http://snomed.info/sct",
+            "code": "386452003"
+          }
+        }
+      ],
+      "relatedResources": [
+        {
+          "resource": "Observation",
+          "searchParameter": "subject",
+          "sortConfigs": [
+            {
+              "paramName": "effectiveDateTime",
+              "dataType": "DATETIME"
+            }
+          ]
+        }
+      ]
+    }
+  ],
```

Comment:
Is the result fo the execution here combined with that of the base and related resources? If no, how do the rules access them?